### DISCUSSION TITLE
docker can't  pull image

### DISCUSSION THREAD

**Discussion by Alyle-L** [qd-today/qd#428] [General]
docker pull qdtoday/qd
Using default tag: latest
latest: Pulling from qdtoday/qd
a71ff8a5264a: Retrying in 1 second 
6f0c0f2f4d6e: Retrying in 1 second 
dfac66dcfd8b: Retrying in 1 second 
8965d06ae626: Waiting 
fc427f7cdafc: Waiting 
60ab5d9e627c: Waiting 
5f8ce4276ef3: Waiting 
error pulling image configuration: download failed after attempts=6: dial tcp 104.18.121.25:443: i/o timeout

**Comment by a76yyyy** [comment]
这个不是框架的问题，docker在国内被墙了，换镜像源

或者使用加速源:
```shell
docker pull dockerproxy.com/qdtoday/qd
```
